packages/console/src/hooks/use-api.ts

```diff
@@ -25,22 +25,28 @@ const toastError = async (response: Response) => {
   }
 };
 
-const useApi = () => {
+type Parameters = {
```

Comment:
nit: Props? Since it should be `properties` instead of `parameters`

Reply:
updated